disable_prelinking.cf

```diff
@@ -1,35 +1,32 @@
 # Policy to disable prelinking
-
-bundle edit_line option_set_no(option)
-{
-  # Set option value to 'no'
-  replace_patterns:
-      "^$(option)=(?!no).*$"
-        replace_with => value("$(option)=no");
-}
-
 bundle agent disable_prelinking
 {
   # Enable autorun
   meta:
+    linux::
       "tags" slist => { "autorun" };
 
+  vars:
+      "_config_file"
+        string => ifelse( "suse|redhat", "/etc/sysconfig/prelink",
+                          "debian", "/etc/default/prelink",
```

Comment:
This line is not strictly needed, as the else case below has the exact same outcome

Reply:
Yes, `ifelse()` requires an odd number of arguments. I explicitly listed debian but you could omit it.